internal/proxy/proxy_test.go

```diff
@@ -1545,7 +1545,7 @@ func TestProxy(t *testing.T) {
 			Type:           milvuspb.ShowType_InMemory,
 		})
 		assert.NoError(t, err)
-		assert.Equal(t, commonpb.ErrorCode_Success, resp.Status.ErrorCode)
+		assert.Equal(t, commonpb.ErrorCode_UnexpectedError, resp.Status.ErrorCode)
```

Comment:
Change UnexpectedError to ErrorCode_Unknown?

Reply:
Modifying common proto requires too much code to change in all components, and if it is really necessary, it is recommended to make a separate pr.